src/renderer/components/RenderMedia.js

```diff
@@ -0,0 +1,29 @@
+const React = require('react')
+
+class RenderMedia extends React.Component {
+  render () {
+    var { className, url, contentType } = this.props
+    if (!contentType) contentType = 'image/jpg'
+    let elm = <div />
+    // TODO: there must be a stable external library for figuring out the right
+    // html element to render
+    if (contentType) {
```

Comment:
This if statement can be removed since it's set to `'image/jpg'` above (or _that_ if statement should be removed).